config/webpack/environment.js

```diff
@@ -62,4 +62,15 @@ Object.assign(fileLoader.use[0].options, {
   postTransformPublicPath: (p) => `window.rails_asset_host + ${p}`
 });
 
+environment.config.merge({
+  optimization: {
+    splitChunks: {
+      chunks (chunk) {
+        // exclude dev portal packs
+        return !['stats', 'active_docs', 'load_stripe', 'validate_signup'].includes(chunk.name)
```

Comment:
Why are these excluded? Can you maybe put a comment here explaining what this config does?

Reply:
It excludes dev portal packs from being chunked, because they're loaded with `javascript_pack_tag` in the liquid templates.
https://github.com/3scale/porta/blob/850a6a2b7f73a14c6dce7e20c2d7e68a72e2b214/lib/developer_portal/lib/liquid/filters/rails_helpers.rb#L14

Reply:
Added a more verbose comment and a link to rails_helper.rb

Reply:
Thanks!

Reply:
OK I figured it out and they can be included 😄